strawberry/scalars.py

```diff
@@ -1,17 +1,16 @@
-from datetime import date, datetime, time
-from decimal import Decimal
-from typing import Any, NewType
-from uuid import UUID
+from typing import Any, Dict, NewType, Union
 
+from .custom_scalar import ScalarDefinition, ScalarWrapper
 
-ID = NewType("ID", str)
-SCALAR_TYPES = [int, str, float, bytes, bool, ID, UUID, datetime, date, time, Decimal]
 
+ID = NewType("ID", str)
 
-def is_scalar(annotation: Any) -> bool:
-    type_ = getattr(annotation, "__supertype__", annotation)
 
-    if type_ in SCALAR_TYPES:
+def is_scalar(
+    annotation: Any,
+    scalar_registry: Dict[object, Union[ScalarWrapper, ScalarDefinition]],
+) -> bool:
+    if annotation in scalar_registry:
```

Comment:
we should probably move this to the schema converter, not sure it makes a lot of sense to keep it here now that we have to pass the scalar_registry